### PR TITLE
Fixes for issues #3286, #3663, Partner Page SVG Transitions, and Catalog Page Transitions

### PR DESCRIPTION
**Pull request by randychilau**
**Description**

This PR fixes:
#3286 
#3663 
Partner Page SVG transition Issues
Catalog Page Dark Mode transition issues
Clearing dev warnings

---

PR 3286 Smoothly displaying the answers in the FAQ on page /careers/programs

Fix is [credited to this post](https://github.com/springload/react-accessible-accordion/issues/356#issuecomment-1309206331).

before:

[accordian_snap.webm](https://github.com/layer5io/layer5/assets/90356410/5bba010d-8f46-4fc2-bc81-1acedb3bcf22)

with fix:

[accordian_fix (1).webm](https://github.com/layer5io/layer5/assets/90356410/51f3f8e6-c601-46de-83aa-8e64b9ebd4dd)

---

PR 3663  Dark-mode: Unable to see text highlight on selection of text

Fix is [credited to this post](https://stackoverflow.com/questions/71567352/how-do-you-fix-the-darkened-text-selection-color-in-safari-css)

before:

[safari_selection.webm](https://github.com/layer5io/layer5/assets/90356410/a2b18914-171b-4f19-8196-a2fa69c1eedf)

with fix:

[text_selected_safari.webm](https://github.com/layer5io/layer5/assets/90356410/affcaf7b-345c-4915-9262-45879181e9f3)

---
Partner Page SVG Issues: 
- needed responsive Citrix SVG, 
- needed responsive Metabit SVG,
- missing semicolon in styled-component breaking SVG transitions, 
- text color transition not in sync

before:

[partner_issues.webm](https://github.com/layer5io/layer5/assets/90356410/03d3e3fd-45b9-42bf-b56d-bf85c1c4b8b9)

after:

[partner_fixed.webm](https://github.com/layer5io/layer5/assets/90356410/eb22fbc8-8612-45e5-ac8b-78a2f003fca8)

---

Catalog Page transition issues:
- section background and text not in sync
- slider background and card text not in sync

before:

[catalog_before.webm](https://github.com/layer5io/layer5/assets/90356410/cc025f27-d564-4e5c-a9f9-a125867cbf03)

after:

[catalog_after.webm](https://github.com/layer5io/layer5/assets/90356410/c5628292-e5ca-4e25-be99-9af74650b507)

---

Cleared the following dev warnings:

`warn Component ResourceWrapper was not imported, exported, or provided by MDXProvider as global scope`
`warn Component ChapterStyle was not imported, exported, or provided by MDXProvider as global scope`

---

**Notes for Reviewers**

Please feel free to let me know if I have to break this into separate PRs or remove a portion of this PR for whatever reasons.

Regarding the issues 3268, 3663, I was looking to fix older issues that have gone stale or were

- older than 5 months
and either
  - no assignee
  or
  - no activity by assignee for over two months

---

I have added comments to other similar older issues that perhaps needed some additional clarification to help reach `closed` status:

- Issue 3430 -  [White Logo] Need a new white logo for kubewarden 
- Issue 3598 - Icons missing for emissary-ingress and NGINX
- Issue 3627 - Add cache policy for images 
- Issue 4212 - Ondat Operator Logo Invisible in light mode 

---

Also note, the [package-lock.json](https://github.com/layer5io/layer5/pull/4296/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) has been updated for a more recent version of a [custom package](https://github.com/layer5labs/gatsby-plugin-meta-redirect) from the Layer5 Labs account. The newer version removes console.logs that were used for testing in the build process.

---

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b650fe4c9a3b618e5448c6a09b8e870da179aac2 at: https://6470575b8de3c956ac164239--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@randychilau Let's discuss this on websites call, I'll add this as an agenda item in [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by Shivam-AfA** [comment]
@randychilau Is this PR ready from your side?

**Comment by randychilau** [comment]
Hi @Shivam-AfA,

Thanks for the message and checking the status of this PR.

I forgot to mention in the Notes that the [package-lock.json](https://github.com/layer5io/layer5/pull/4296/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) file has been updated and should be included with the merge. 

There is a version change of a Layer5 Lab [package](https://github.com/layer5labs/gatsby-plugin-meta-redirect). The new version removes console.logs that were used for testing in the build process.

Thanks again,
Randy

**Comment by randychilau** [comment]
Note: Also added a commit to fix an eslint warning on spacing that I missed in [this file](https://github.com/layer5io/layer5/pull/4319/files#diff-cd44e26e933c415bce93c0e45e13687e8c70ce07b93cdb1601435a097d341a0b) on a recently merged PR.

image

Thanks!

**Comment by l5io** [comment]
🚀 Preview for commit 8a697942867bc6f5d8009d77ce82c1f54daa97ab at: https://647b738a1de38023e96b0e18--layer5.netlify.app

**Comment by goldfishdolphin** [comment]
Great work @randychilau 👍🏽

**Comment by randychilau** [comment]
fyi: this PR was discussed on the Websites Weekly Call May 29th, 2023 ([timestamp: 14.57](https://www.youtube.com/live/D-pGXxyfwuY?feature=share&t=897)).

happy to answer any other questions on this.

Cheers,
Randy

**Comment by l5io** [comment]
🚀 Preview for commit 481e59cb6aa75ded80c24c096d535a071d0aa015 at: https://647e5dcc1991352a6108f1da--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1cc4fe20627f401fc6bac818747e7711e180125e at: https://64875e7a9f501a07ff06b177--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
This is a great PR @randychilau. Your documentation and explanation skills are a delight. Thanks for these contributions. 🚀

File: src/collections/resources/articles/terraform-with-meshery/index.mdx [hunk 1]
```diff
@@ -30,23 +30,50 @@ import Button from "../../../../reusecore/Button";
 </div>
 
 <p>
-    One way to further enhance your use of Terraform is by integrating it with Meshery. Meshery is a cloud-native management platform that provides a unified interface for managing and monitoring your infrastructure resources, including those managed by Terraform. By integrating Terraform with Meshery, you can leverage the power and flexibility of both tools to streamline your infrastructure management process.
+  One way to further enhance your use of Terraform is by integrating it with
```
Comment (Shivam-AfA) [inline src/collections/resources/articles/terraform-with-meshery/index.mdx:33]: Just a question, why are we modifying this file by puttig line breaks?

**Comment by goldfishdolphin** [comment]
> This is a great PR @randychilau. Your documentation and explanation skills are a delight. Thanks for these contributions. 🚀

I agree with @Shivam-AfA.
It’s really impressive how @randychilau document everything so well.

File: src/collections/resources/articles/terraform-with-meshery/index.mdx (hunk 1 above)
Comment (randychilau) [reply src/collections/resources/articles/terraform-with-meshery/index.mdx:33]: Hi @Shivam-AfA,

Thanks for your message and working on this PR.

This file was mainly edited to update the `<ResourceWrapper>`  tag to `<ResourcesWrapper>`
image

to match the import statement: 
`import { ResourcesWrapper } from "../../Resources.style.js";` 

and clear a MDX warning that was appearing during builds:
`warn Component ResourceWrapper was not imported, exported, or provided by MDXProvider as global scope`

Not sure if I also made edits to the spacing as part of general page cleanup/tidying.

Feel free to let me know if you would like me to remove the line break edits.

Thanks,
Randy

**Comment by l5io** [comment]
🚀 Preview for commit 481f0331df9ec6c5d18ea8268862dfa6cf141a20 at: https://648e97bfcc1ae5769516a1d5--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
@randychilau I removed the line breaks in the blog. Merging this PR...

**Comment by randychilau** [comment]
Hi @Shivam-AfA,

Thanks for taking the time to review and merge the PR!

Cheers,
Randy

**Pull request merged**